index.js

```diff
@@ -31,7 +31,7 @@ class Dat extends EventEmitter {
     const discoveryOpts = Object.assign({
       stream: (info) => this._replicate(info)
     }, this.opts)
-    this.swarm = new DiscoverySwarmWeb(discoveryOpts)
+    this.swarm = DiscoverySwarmWeb(discoveryOpts)
```

Comment:
Was there an error popping up when you had the new keyword here?

Reply:
Yup, if we want to use the `new` keyword we have to do a named import. I think I can update discovery-swarm-web docs to highlight that.

Reply:
Interesting. Docs for that would be handy to have. :)